### PR TITLE
Add cluster-admin privileges to MGO's ServiceAccount

### PR DESCRIPTION
**Pull request by csheremeta**
Clarify instructions in README, example CR files, etc to note that SA is required
This resolves OSD-5236 and OSD-4941

### CODE REVIEW

**Comment by csheremeta** [comment]
/hold
pending discussion with Dustin

File: deploy/05_must-gather-admin.ClusterRoleBinding.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: must-gather-admin
+subjects:
+- kind: ServiceAccount
+  name: must-gather-admin
+￼  namespace: openshift-must-gather-operator
```
Comment (dustman9000) [inline deploy/05_must-gather-admin.ClusterRoleBinding.yaml:8]: Looks like some spacing is off here

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 2]
```diff
@@ -124,6 +133,46 @@ objects:
...
         name: must-gather-operator
         apiGroup: rbac.authorization.k8s.io
+    - apiVersion: rbac.authorization.k8s.io/v1
+      kind: ClusterRoleBinding
+      metadata:
+        name: must-gather-admin
+      subjects:
+      - kind: ServiceAccount
+        name: must-gather-admin
+      "\uFFFC  namespace": openshift-must-gather-operator
```
Comment (dustman9000) [inline hack/olm-registry/olm-artifacts-template.yaml:143]: It looks like some unicode slipped in here due to the spacing issues above...

**Pull request merged**